src/test/java/seedu/address/model/appointment/TimeTest.java

```diff
@@ -16,9 +16,9 @@ public void constructor_null_throwsNullPointerException() {
 
     @Test
     public void testStringConversion() {
-        assertEquals("9:00AM",
-                new Time(LocalTime.parse("09:00")).toString());
-        assertEquals("11:59PM",
-                new Time(LocalTime.parse("23:59")).toString());
+        assertEquals("9:00am",
+                new Time(LocalTime.parse("09:00")).toString().toLowerCase());
+        assertEquals("11:59pm",
+                new Time(LocalTime.parse("23:59")).toString().toLowerCase());
```

Comment:
Don't change this because the time is supposed to be displayed as "9:00AM" and not "9:00am"

Reply:
It is strange that it is displayed as "9:00am" on my laptop. maybe due to java versions?

Reply:
Restored